lib/array.js

```diff
@@ -60,6 +60,32 @@ helpers.before = function(array, n) {
   return array.slice(0, -n);
 };
 
+/**
+ * Return all of the items in the two collections specified in
+ * a single collection.
+ *
+ * ```handlebars
+ * <!-- foo: ['a', 'b'] -->
+ * <!-- bar: ['c', 'd'] -->
+ * {{concat foo bar}}
+ * <!-- results in: '["a", "b", "c", "d"]' -->
+ * ```
+ * @param {Array} `arrayA`
+ * @param {Array} `arrayB`
+ * @return {Array} Array containing first the elemens of array As then of B
+ * @api public
+ */
+
+helpers.concat = function(arrayA, arrayB) {
+  if (Array.isArray(arrayA) && !Array.isArray(arrayB)) {
```

Comment:
Can I suggest an implementation that would handle more than one array? Something like:
```
    const args = Array.from(arguments);
    args.pop(); // Pop off options.
    return [].concat(...args)
```

Reply:
That seems doable and a good idea. I'm not sure if functionality like that is desired by the maintainers though, so I will not implement it just yet 🙂 

Just for reference, if the current implementation stays you could always do `(concat array1 (concat array2 array3))`. But of course this may become hard to read for a larger number of arrays.